src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public Task(String description) {
```

Comment:
For statements of different types, will it be better to leave a blank line in between?